src/hydra-queue-runner/build-remote.cc

```diff
@@ -66,7 +70,7 @@ static void openConnection(Machine::ptr machine, Path tmpDir, int stderrFD, Chil
 
         execvp(argv.front().c_str(), (char * *) stringsToCharPtrs(argv).data()); // FIXME: remove cast
 
-        throw SysError("cannot start ssh");
+        throw SysError(format("cannot start %1%") % pgmName);
```

Comment:
BTW it's shorter to write `throw SysError("cannot start %s", pgmName)`.

Reply:
Thx, applied.